lib/dotcom_web/components/gtfs.ex

```diff
@@ -0,0 +1,188 @@
+defmodule DotcomWeb.Components.Gtfs do
```

Comment:
As long as this yields the correct icons, then this comment is non-blocking, but...

If I'm reading this correctly, doesn't this re-implement a lot of `&Routes.Route.icon_atom/1`?

It strikes me as though most of the logic in this file might be easier to follow if it passed the route into `icon_atom`, and then used the result to actually render the icon. I _think_ that would make the pattern-matching easier.


---

Unrelated, but another comment at this same code-location: I don't think I like the name `Gtfs` for this module. While routes (and route icons/symbols) are related to GTFS, I think that they are fundamentally different concepts. GTFS is the data format used to transit route data, while routes are their own thing, whether communicated through GTFS, the V3 API, or carrier pigeon. 

Also, related to our chat earlier today, while I do sort of get why Phoenix crams all of its "core" components into one `core_components.ex` file, I don't think it's necessary to do that for the rest of our stuff. If it's easier to manage by having a separate file per component, or a separate file per class-of-component, or whatever, then I'm all for separating things out. All that to say - I don't think we need these components to live under a wide umbrella for right now.

What about calling this module `RouteSymbols` or `RouteIcons`?